obspy/io/sac/tests/test_sacpz.py

```diff
@@ -63,6 +63,15 @@ def test_write_sacpz_multiple_channels(self):
         expected = [l for l in expected.split("\n") if "CREATED" not in l]
         self.assertEqual(got, expected)
 
+    def test_write_sacpz_soh(self):
+        path = os.path.join(self.path, '..', '..', 'stationxml', 'tests',
+                            'data', 'only_soh.xml')
+        inv = read_inventory(path)
+        f = io.StringIO()
+        inv.write(f, format='SACPZ')
```

Comment:
This triggers two (expected) warnings. They need to be properly caught and tested/asserted. (Look for `catch_warnings(...)` in our test files)